pages/api/leaderboardAPI/competition/[id].tsx

```diff
@@ -0,0 +1,28 @@
+import mongoose from "mongoose";
+import { NextApiRequest, NextApiResponse } from "next";
+import { ScoreModel } from "models/leaderboardSchema";
+import { url } from "pages/api/_db";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { method, query } = req;
+
+  await mongoose.connect(url);
+
+  switch (method) {
+    case "GET": {
+      res
+        .status(200)
+        .json(
+          await ScoreModel.find({ CID: query })
+            .populate("UID")
+            .populate("CID")
```

Comment:
you can populate multiple with just `.populate("UID CID")`

Reply:
Thank you, didn't know that! :)